lib/main_containers.pm

```diff
@@ -312,6 +312,7 @@ sub load_container_tests {
             load_host_tests_containerd_crictl() if (/containerd_crictl/i);
             load_host_tests_containerd_nerdctl() if (/containerd_nerdctl/i);
             loadtest('containers/kubectl') if (/kubectl/i);
+            loadtest('containers/kubeadm') if (/kubeadm/i);
```

Comment:
aren't we just replacing the test or this is a mistake?

Reply:
Fixed.